File: C#/0026. Remove Duplicates from Sorted Array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        if (nums==null || nums.Length==0)
            return 0;
        int cur = 0;
        for (int i = 1; i < nums.Length; i++)
        {
            if (nums[cur] != nums[i])
            {
                cur++;
                nums[cur] = nums[i];
            }
        }
        return cur+1;
    }
}

File: archives/leetcode2/0026. Remove Duplicates from Sorted Array.py
class Solution:
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if not nums:
            return 0
        if len(nums)==1:
            return 1
        if len(nums)>1:
            left=0
            right=1
            for right in range(1,len(nums)):
                if nums[right]!=nums[left]:
                    nums[left+1],nums[right]=nums[right],nums[left+1]
                    right+=1
                    left+=1
            return left+1
        
                
